setup.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from setuptools import Command, setup
+from setuptools import find_packages
+from unittest import TestLoader, TextTestRunner
+
+from os import environ, path
+
+PROJECT = 'oca-decorators'
+SHORT_DESC = (
+    'This is a library of decorators for Odoo developers.'
+)
+README_FILE = 'README.rst'
+
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: GNU Lesser General Public License v3 or '
+    'later (LGPLv3+)',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+
+version = environ.get('RELEASE') or environ.get('VERSION') or '0.0.1'
```

Comment:
Shouldn't the package have a `__version__` attribute to read here? 🤔

Reply:
Got me dude, I made this all up. I put the version in the Travis file. Where should I be putting it?